MonoGame.Framework/Graphics/States/CompareFunction.cs

```diff
@@ -1,14 +1,45 @@
-﻿namespace Microsoft.Xna.Framework.Graphics
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+namespace Microsoft.Xna.Framework.Graphics
 {
+    /// <summary>
+    /// Defines comparison functions for the various rendering tests.

```

Comment:
Maybe instead of saying "various rendering tests" you should specify the cases like:  ```The comparison function for the depth, stencil, and alpha.```.